android/automation.md

```diff
@@ -0,0 +1,175 @@
+
+* [Chain of Trust](http://scriptworker.readthedocs.io/en/latest/chain_of_trust.html)
+* [pushapkscript](https://github.com/mozilla-releng/pushapkscript)
+
+### 📓 Planned: Upload to archive.mozilla.org
+
+We want to publish every release on archive.mozilla.org. Currently we upload those builds manually. In automation this will be done by  the beetmover_scriptworker.
+
+Before we can automate this task we need to automate signing builds and have a chain of trust.
+
+* [beetmover script](https://github.com/mozilla-releng/beetmoverscript/)
+* [beetmover configuration for Fennec](https://github.com/mozilla-releng/beetmoverscript/blob/master/beetmoverscript/templates/fennec_nightly.yml)
+
+### 📓 Planned: Release signoff
+
+This is a dummy task that does nothing except waiting on somebody to complete it. After the build is tested and the release stakeholders agree to ship, someone from the Focus team will use the taskcluster-cli tool to manually resolve this task.
+
+* [taskcluster ci](https://github.com/taskcluster/taskcluster-cli)
+
+### 📓 Planned: Publish
+
+Finally we want to publish a release build on Google Play
+
+Like other scriptworker instances, we can use the existing instance. The current implementation is fennec-specific. We have to add support there too. pushapk_scriptworker mainly delegates checks and uploads to mozapkpublisher. That tool needs Focus support. Working on mozapkpublisher doesn't require any Taskcluster knowledge.
+
+* [pushapkscript](https://github.com/mozilla-releng/pushapkscript)
+* [mozapkpublisher](https://github.com/mozilla-releng/mozapkpublisher)
+
+## Nightly (Alpha) pipeline
+
+We want to build Nightly versions from master. Those Nightly builds should be distributed via the Google Play Alpha channel and replace distributing via buddybuild. Updates via buddybuild require manual download and installation of updates. With Google Play builds can update automatically in the background.
+
+The Alpha channel is not intended to be public like the Beta channel. However it should be open to the L10N teams and interested contributors.
+
+The pipeline for Nightly builds looks like the release sign-off pipeline without the upload and sign-off tasks.
+
+* 📓 Planned: Build (triggered via taskcluster hook)
+* 📓 Planned: Signing
+* 📓 Planned: Upload to Google Play
+* 📓 Planned: Publish on Alpha channel
+
+## Maintenance tasks
+
+### ✅ Active: String import
+
+Every day at 12:00 UTC we run a script that imports the latest translations from the L10N repository and creates a pull request for the import if there are new strings. Eventually this import can be merged automatically. For now we want to review those changes manually to validate that the task is running as intended.
+
+* [Task script](https://github.com/mozilla-mobile/focus-android/blob/master/tools/taskcluster/import_strings_and_create_pull_request.sh)
+* [Hook and task definition](https://tools.taskcluster.net/hooks/project-focus/strings-import)
```

Comment:
That's how we do in in Focus and Chenxia use the same task for firefox-tv. However we will have our new L10N process ready soon.

Reply:
I made a note that we want to improve this process.